src/coq_elpi_builtins.ml

```diff
@@ -837,17 +836,11 @@ let add_axiom_or_variable api id sigma ty local =
     err Pp.(str"coq.env.add-const: the type must be ground. Did you forge to call coq.typecheck-indt-decl?");
   let gr, _ =
     if local then begin
-      let uctx =
-        let context_set_of_entry = function
-          | Entries.Polymorphic_entry uctx -> Univ.ContextSet.of_context uctx
-          | Entries.Monomorphic_entry uctx -> uctx in
-        context_set_of_entry uentry in
-      DeclareUctx.declare_universe_context ~poly:false uctx;
```

Comment:
Not sure why `poly` was always `false` here

Reply:
that's on me, since Coq-Elpi only supports universe monomorphic stuff

Reply:
Hum, indeed @herbelin where is the poly flag now?

Reply:
it may, in the future. So I guess it is fine for now.

Reply:
The old code was taken from something in vernac/, so it was probably handling also the poly case, but was dead code really.